tests/unit/core/TestInstallAddon.js

```diff
@@ -3,6 +3,7 @@ import React from 'react';
 import { renderIntoDocument } from 'react-addons-test-utils';
 
 import createStore from 'amo/store';
+import { setInstallState } from 'core/actions/installations';
```

Comment:
Just realised this is called `setInstallState` but I think going forward we should use `reducerNameActionVerb` as our template for actions. For instance we do `categoriesFetch`, `categoriesLoad`, `categoriesFail`. I think this helps us identify the type of action much better and while it's a bit awkward from an English-speaker's perspective re: word-order I think it's the best way to make things consistent and easily grokable.

E.g. I think this should be `installStateSet`. Or even `installSet`.

Fwiw I used `currentViewSet` for a similar action in #2514 and it felt natural.